docs/src/content/hardhat-runner/docs/guides/getting-help.md

```diff
@@ -1,3 +1,32 @@
 # Getting help
 
-\<New guide, TODO>
+Hardhat has a strong community of users willing to help you in times of trouble. Please, read this guide entirely to learn where and how to ask for help more effectively.
+
+The first place to look for answers is the [GitHub Discussions section in the Hardhat repository.](https://github.com/nomiclabs/hardhat) We recommend you search there first, as the answer may already exist.
+
+If you can't find what you are looking for on GitHub Discussions, you can create a new Discussion.
+
+If you weren't lucky on GitHub, or you prefer a real-time chat, you can join our [Discord Server](https://hardhat.org/discord). Please, read its rules, and ask for help in the right channel.
+
+## Asking an effective question
+
+To get an answer quickly on GitHub Discussions and Discord, you need to make sure you write a good question first. To do so, you need to include:
+
+1. A clear description of what you are trying to do.
+2. The results you are getting, and how they differ from what you expect.
+3. Which version of Hardhat you are running, and which plugins and their versions you are using.
+4. If possible, a link to a public repository to reproduce your problem.
+
+## Reporting a bug
+
+If you think you found a bug on Hardhat, and you didn't get help by following this guide, we'd appreciate if you [report it](https://github.com/NomicFoundation/hardhat/issues).
+
+Please, follow these steps before opening a new issue:
+
+1. Make sure you are using the latest version of Hardhat and its plugins. Your problem may already be fixed.
+2. Try to identify if the issue comes from a plugin by running your project with some of them disabled. We only accept issues about plugins starting with `@nomiclabs/` and `@nomicfoundation/`.
+3. Use the search on GitHub to try to find other reports of the same problem. If you find one, please comment on the existing issue instead of creating a new one.
```

Comment:
```suggestion
3. Use the search on GitHub to try to find other reports of the same problem. If you find one and you can add useful new information, please comment on the existing issue instead of creating a new one.
```

Reply:
@fvictorio I respectfully disagree with this suggested change. It's useful for us to know that more people are encountering the problem, even if those additional people don't have anything constructive to add.

Reply:
That's fair, I just don't want to encourage "+1" comments. I thought about suggesting reacting with a :+1:, but I couldn't find a link to some canonical explanation on upvoting issues with reactions.

Reply:
Unfortunately `+1` is useful. Annoying, but useful. I'd blame GitHub and not the users for that.